onnx/common/ir_pb_converter.cc

```diff
@@ -6,7 +6,7 @@
 namespace ONNX_NAMESPACE {
 
 // Part 1: convert ONNX Protobuf to IR
-std::unique_ptr<Graph> graphProtoToGraph(const ONNX_NAMESPACE::GraphProto& gp);
+std::unique_ptr<Graph> graphProtoToGraph(const ONNX_NAMESPACE::GraphProto& gp, bool nested=false);
```

Comment:
since this isn't a public api and has like 1 or 2 callsites, i'd slightly prefer to not have the parameter defaulted.